constraints/miden-vm/stack.air

```diff
@@ -0,0 +1,957 @@
+        swapw2([stack])                        when op_flags[28]
+        swapw3([stack])                        when op_flags[29]
+        swapdw([stack])                        when op_flags[30]
+
+        assert([stack])                        when op_flags[32]
+        eq([stack], [h0])                      when op_flags[33]
+        add([stack])                           when op_flags[34]
+        mul([stack])                           when op_flags[35]
+        and([stack])                           when op_flags[36]
+        or([stack])                            when op_flags[37]
+        u32and([stack], [b_chip])              when op_flags[38]
+        u32xor([stack], [b_chip])              when op_flags[39]
+        # TODO: add match variant for caller 
+        frie2f4()
+        drop([stack])                          when op_flags[41]
+        cswap([stack])                         when op_flags[42]
+        cswapw([stack])                        when op_flags[43]
+        mloadw([stack], [clk, b_chip])         when op_flags[44]
+        mstore([stack, helper, clk], [b_chip]) when op_flags[45]
+        mstorew([stack], [clk, b_chip])        when op_flags[46]
+        fmpupdate([stack, fmp])                when op_flags[47]
+
+        pad([stack])                           when op_flags[48]
+        dup([stack])                           when op_flags[49]
+        dup1([stack])                          when op_flags[50]
+        dup2([stack])                          when op_flags[51]
+        dup3([stack])                          when op_flags[52]
+        dup4([stack])                          when op_flags[53]
+        dup5([stack])                          when op_flags[54]
+        dup6([stack])                          when op_flags[55]
+        dup7([stack])                          when op_flags[56]
+        dup9([stack])                          when op_flags[57]
+        dup11([stack])                         when op_flags[58]
+        dup13([stack])                         when op_flags[59]
+        dup15([stack])                         when op_flags[60]
+        advpop([stack])                        when op_flags[61]
+        sdepth([stack, bookkeeping])           when op_flags[62]
+        clk([stack], [clk])                    when op_flags[63]
+
+        u32add([stack, helper])                when op_flags[64]
+        u32sub([stack, helper])                when op_flags[65]
+        u32mul([stack, helper])                when op_flags[66]
+        u32div([stack, helper])                when op_flags[67]
+        u32split([stack, helper])              when op_flags[68]
+        u32assert2([stack, helper])            when op_flags[69]
+        u32add3([stack, helper])               when op_flags[70]
+        u32madd([stack, helper])               when op_flags[71]
+
+
+# No stack shift operations
+
+ev noop(main: [s[16]]):
+    enf elem' = elem for elem in s
```

Comment:
Since we are doing it everywhere else, we can probably use indexed access here as well. For example:
```
ev noop(main: [s[16]]):
    enf s[i]' = s[i] for i in 0..16
```